pom.xml

```diff
@@ -42,6 +42,7 @@
     <docker.tag.long>${docker.tag.prefix}9.${jetty9.minor.version}-${maven.build.timestamp}</docker.tag.long>
 
     <docker.openjdk.image>openjdk:8</docker.openjdk.image>
+    <gcloud-projectId-file>${project.build.directory}/gcloud-projectid.properties</gcloud-projectId-file>
```

Comment:
Fair enough, fix will arrive on a new issue / PR.
This PR is holding up too many other PRs at the moment.